hack/minikube-rook-mirror-setup.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+set -x
+set -e -o pipefail
+
+## Variables
+PRIMARY_CLUSTER="${PRIMARY_CLUSTER:-hub}"
+SECONDARY_CLUSTER="${SECONDARY_CLUSTER:-cluster1}"
+POOL_NAME="replicapool"
+
+## Usage
+usage()
+{
+	set +x
+	echo "Usage:"
+	echo "  $0"
+	echo "  Available environment variables:"
+	echo "    minikube primary cluster PRIMARY_CLUSTER ${PRIMARY_CLUSTER}"
+	echo "    minikube secondary cluster SECONDARY_CLUSTER ${SECONDARY_CLUSTER}"
+	exit 0
+}
+
+function wait_for_condition() {
+    local count=15
+    local condition=${1}
+    local result
+    shift
+
+    while ((count > 0)); do
+        result=$("${@}")
+        if [[ "$result" == "$condition" ]]; then
+            return 0
+        fi
+        count=$((count - 1))
+        sleep 5
+    done
+
+    echo "Failed to meet $condition for command $*"
+    exit 1
+}
+
+SECONDARY_CLUSTER_PEER_TOKEN_SECRET_NAME=$(kubectl get cephblockpools.ceph.rook.io "${POOL_NAME}" --context="${SECONDARY_CLUSTER}" -nrook-ceph -o jsonpath='{.status.info.rbdMirrorBootstrapPeerSecretName}')
+SECONDARY_CLUSTER_SECRET=$(kubectl get secret -n rook-ceph "${SECONDARY_CLUSTER_PEER_TOKEN_SECRET_NAME}" --context="${SECONDARY_CLUSTER}" -o jsonpath='{.data.token}'| base64 -d)
```

Comment:
This command failed for me:
```
+ PRIMARY_CLUSTER=hub SECONDARY_CLUSTER=cluster1 ramen/hack/minikube-rook-mirror-setup.sh
+ set -e -o pipefail
+ PRIMARY_CLUSTER=hub
+ SECONDARY_CLUSTER=cluster1
+ POOL_NAME=replicapool
++ kubectl get cephblockpools.ceph.rook.io replicapool --context=cluster1 -nrook-ceph -o 'jsonpath={.status.info.rbdMirrorBootstrapPeerSecretName}'
+ SECONDARY_CLUSTER_PEER_TOKEN_SECRET_NAME=
++ kubectl get secret -n rook-ceph '' --context=cluster1 -o 'jsonpath={.data.token}'
++ base64 -d
error: resource name may not be empty
+ SECONDARY_CLUSTER_SECRET=
+ echo exit value: 1
```
Subsequent runs were successful.  Initial run failed, perhaps because it was performed shortly after `minikube-rook-setup.sh`.  One solution is to wait for the resource to exist.

Reply:
We now wait at the end minikube-rook-setup.sh for the secret reference in the pool status, before proceeding. Should address this issue.